### PR TITLE
Fixed visit recreation on cookie timeout (#174)

### PR DESCRIPTION
**Pull request by ex0rcist**

### CODE REVIEW

**Comment by ankane** [comment]
Thanks Eugene, this looks great. I've added a few comments inline on how to make the diff more readable.

File: vendor/assets/javascripts/ahoy.js [hunk 1]
```diff
@@ -98,7 +98,11 @@
   }
 
   function trackEvent(event) {
-    ready( function () {
+    if (!ahoy.getVisitId()) {
+      ready(createVisit)
+    }
+
+    ready(function () {
```
Comment (ankane) [inline vendor/assets/javascripts/ahoy.js]: Same line as above, but missing space.

File: vendor/assets/javascripts/ahoy.js [hunk 2]
```diff
@@ -137,65 +141,69 @@
...
+        $.post(visitsUrl, data, setReady, "json");
+      } else {
+        log("Cookies disabled");
+        setReady();
+      }
     }
   }
 
-  ahoy.getVisitId = ahoy.getVisitToken = function () {
-    return visitId;
```
Comment (ankane) [inline vendor/assets/javascripts/ahoy.js]: Is there a strong reason to move these functions?

**Comment by ex0rcist** [comment]
Thanks Andrew, my bad
1) moving this functions was caused with calling createVisit() before they are defined. fixed
2) space - fixed

**Comment by ankane** [comment]
Thanks for tracking down and fixing! 💯

**Pull request merged**